scss/components/_breakdowns.scss

```diff
@@ -1,3 +1,18 @@
+.breakdown-container {
+  @include clearfix();
+  padding: u(0 2rem);
+  position: relative;
+  width: 100%;
+
+  .sidebar-container {
```

Comment:
This and the `.main__content--right` selectors should be within a media query so that it stacks full width on mobile.